### PR TITLE
Customize admin favicon using admin.icon_image setting

### PR DESCRIPTION
**Pull request by olegts239**
Adding possibility to change favicon using admin.icon_image setting

### CODE REVIEW

**Review by MrCrayon — CHANGES_REQUESTED** [review]
I think in this case a one line solution might be enough:
```diff
-<link rel="shortcut icon" href="{{ voyager_asset('images/logo-icon.png') }}" type="image/x-icon">
+<link rel="shortcut icon" href="{{ Voyager::setting('admin.icon_image') ? Voyager::image(Voyager::setting('admin.icon_image')) : voyager_asset('images/logo-icon.png') }}" type="image/x-icon">
```
But if you want to keep the if else for clarity, to not repeat html, I would do something like this:
```blade
@php
if (Voyager::setting('admin.icon_image')) {
    $admin_favicon = Voyager::image(Voyager::setting('admin.icon_image'));
} else {
    $admin_favicon = voyager_asset('images/logo-icon.png');
}
@endphp
<link rel="shortcut icon" href="{{ $admin_favicon }}" type="image/x-icon">
```

**Comment by olegts239** [comment]
@MrCrayon thanks for your comment, i used the same code in my commit as [there](https://github.com/the-control-group/voyager/blob/e3ab444f365eaf6ef140004737a4fe6aa24c6b3e/resources/views/master.blade.php#L58-L63) for unified code styling

**Review by MrCrayon — DISMISSED** [review]
@olegtsoy I saw about your choice for code styling, it makes sense.

Works for me.

**Comment by emptynick** [comment]
I'm not a hundred percent sure that this is compatible with
https://github.com/the-control-group/voyager/blob/317ce0f2014b08ea4f0f3ead0f973f1c952f642c/resources/views/login.blade.php#L50-L55
What happens if you upload a .ico file? Also, I think some browser don't directly accept a PNG as a favicon (IE for example)

**Comment by MrCrayon** [comment]
According to https://en.wikipedia.org/wiki/Favicon PNG and ICO are supported on all modern browsers, including IE11.
If someone is still using older IE versions not seeing bookmark icon might be the last of their problems :laughing:

**Comment by emptynick** [comment]
Its more about ` type="image/x-icon"`.
Uploading a PNG might (not sure) result in errors/notices in the console.
Can't we just change it to `image/png` and only use `image/x-icon` for the default .ico file?

File: resources/views/master.blade.php [hunk 1]
```diff
@@ -10,7 +10,12 @@
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700" rel="stylesheet">
 
     <!-- Favicon -->
-    <link rel="shortcut icon" href="{{ voyager_asset('images/logo-icon.png') }}" type="image/x-icon">
+    <?php $admin_favicon = Voyager::setting('admin.icon_image', ''); ?>
+    @if($admin_favicon == '')
+        <link rel="shortcut icon" href="{{ voyager_asset('images/logo-icon.png') }}" type="image/x-icon">
```
Comment (fletch3555) [inline resources/views/master.blade.php]: I agree with @emptynick.  Please change to `type="image/png"`

**Review by fletch3555 — CHANGES_REQUESTED** [review]

**Review by emptynick — APPROVED** [review]

**Review by emptynick — APPROVED** [review]

**Pull request merged**